### PR TITLE
Add installation instruction for Astra camera on ROS

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by knorth55** [comment]
i tested and it's ok
astra

File: doc/install_astra_camera.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+----------------------------------------------------
+
+.. code-block:: bash
+
+  sudo apt-get install ros-indigo-openni2-camera ros-indigo-openni2-launch
+
+  cd ~/Downloads/Tools_SDK_OpenNI/OpenNI-Linux-x64-2.2/Samples/Bin
+
+  sudo cp libOpenNI2.so /usr/lib/libOpenNI2.so
+  sudo cp OpenNI2/Drivers/* /usr/lib/OpenNI2/Drives/
```
Comment (wkentaro) [inline doc/install_astra_camera.rst]: typo `Drives` -> `Drivers`

**Comment by wkentaro** [comment]
Please merge this.

**Comment by k-okada** [comment]
please add side view of rviz, so that we can understand the performance of this camera

**Comment by wkentaro** [comment]
> please add side view of rviz, so that we can understand the performance of this camera

@knorth55 Could you please do this?

**Pull request merged**